cli/cmd/plugin/ui/web/tanzu-ui/src/views/unmanaged-cluster/unmanaged-cluster-basic/unmanaged-cluster-network-settings-step/UnmanagedClustersNetworkSettings.test.tsx

```diff
@@ -0,0 +1,39 @@
+// React imports
+import React from 'react';
+
+// Library imports
+import { render, screen, waitFor } from '@testing-library/react';
+
+// App imports
+import UnmanagedClusterNetworkSettings from './UnmanagedClustersNetworkSettings';
+
+jest.mock('react-router-dom', () => ({
+    ...(jest.requireActual('react-router-dom') as any),
+    useNavigate: () => jest.fn(),
+}));
+
+describe('UnmanagedNetworksettings component', () => {
```

Comment:
Nitpick: would change to same name as component 'UnmanagedClusterNetworkSettings'